.github/workflows/tag-publish.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       stage: prd
       target-build: true
       target-lint: true
-      target-unit-tests: true
+      target-tests: true
```

Comment:
This will breaks the pipelines as the "unit-tests" parameter is required by the template